task6/script.js

```diff
@@ -0,0 +1,47 @@
+const df = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  {
+    to: "bookmark.html",
+    img: "1.png",
+    alt: "画像1",
+    text: "ブックマーク"
+  },
+  {
+    to: "message.html",
+    img: "2.png",
+    alt: "画像2",
+    text: "メッセージ"
+  }
+];
+
+const arrayPromise = () => {
+  return new Promise((resolve, reject) => {
+    resolve(listItem);
+  });
+};
+
+arrayPromise().then((value) => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve();
```

Comment:
[IMO]
Since the problem is already solved in line 22, I thought it would be better to just pass it as "then".

Reply:
Thank you for the review.
Correct.